IFC 4.3/Earthworks_01/readme.md

```diff
@@ -0,0 +1,50 @@
+
+## Earthworks\_01
+
+### Intent
+
+This unit test establishes a simple embankment of infrastructure facilities.
+
+### Prerequisites
+
+This scenario builds upon following other scenarios:
+- ProjectSetup_1
+- Tin_1
+
+### Content
+
+This scenario covers the following concepts and/or IFC entities:
+
+- IfcEarthworksFill
+- IfcGeographicElement
+- IfcTriangulatedIrregularNetwork
+- IfcCartesianPointList3D
+- IfcEarthworksFillTypeEnum
+- IfcSite
+- IfcProject
+- IfcOwnerHistory
+- IfcPersonAndOgranization
+- IfcPerson
+- IfcOrganization
+- IfcApplication
+- IfcGeometricRepresentationContext
+- IfcAxis2Placement3D
+- IfcCartesianPoint
+- IfcDirection
+- IfcUnitAssignment
+- IfcSIUnit
+- IfcLocalPlacement
+- IfcRelAggregates
+- IfcProductDefinitionShape
+- IfcShapeRepresentation
+- IfcRelContainedInSpatialStructure

```

Comment:
No need to state these, as they are covered by ProjectSetup_1 and Tin_1.

Reply:
Deleted in a new commit.